raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -243,6 +244,42 @@ def test_init_with_usable_routes():
     assert send_mediated_transfer.recipient == channels[0].partner_state.address
 
 
+def test_init_with_fees_more_than_max_limit():
+    transfer_amount = TokenAmount(100)
+    flat_fee = FeeAmount(int(transfer_amount + MAX_MEDIATION_FEE_PERC * transfer_amount))
+    expected_fee_margin = int(flat_fee * DEFAULT_MEDIATION_FEE_MARGIN)  # == 1
```

Comment:
What does this `== 1` comment mean, here? I don't think the result of that calculation is `1`.

Reply:
It's a remnant from before. I did not add that. Will just remove it.